spanner/transaction_test.go

```diff
@@ -413,19 +413,31 @@ func TestReadWriteTransaction_PrecommitToken(t *testing.T) {
 		query                  bool
 		update                 bool
 		batchUpdate            bool
+		mutationsOnly          bool
 		expectedPrecommitToken string
 		expectedSequenceNumber int32
 	}
 
 	testCases := []testCase{
-		{"Only Query", true, false, false, "PartialResultSetPrecommitToken", 3}, //since mock server is returning 3 rows
-		{"Query and Update", true, true, false, "ResultSetPrecommitToken", 4},
-		{"Query, Update, and Batch Update", true, true, true, "ExecuteBatchDmlResponsePrecommitToken", 5},
+		{"Only Query", true, false, false, false, "PartialResultSetPrecommitToken", 3},
+		{"Query and Update", true, true, false, false, "ResultSetPrecommitToken", 4},
+		{"Query, Update, and Batch Update", true, true, true, false, "ExecuteBatchDmlResponsePrecommitToken", 5},
+		{"Only Mutations", false, false, false, true, "BeginTransactionPrecommitToken", 1},
```

Comment:
Can you add a negative test scenario also?
i.e when a  Commit RPC returns aborted then the retry should set mutation key and the commit request should have latest precommit token
https://github.com/harshachinta/java-spanner/blob/01c03144134febfb3c83e1bc80a7533f53f067ba/google-cloud-spanner/src/test/java/com/google/cloud/spanner/MultiplexedSessionDatabaseClientMockServerTest.java#L1140